### DISCUSSION TITLE
ability to disable map backgrounds

### DISCUSSION THREAD

**Discussion by Floatyoats** [ppy/osu#26481] [Ideas]
On stable, you can disable seasonal backgrounds and I was wondering if we could have a similar feature for map backgrounds like turning them into a black background or something

**Comment by cihe13375** [comment]
On stable, one could delete or change the background file in the song folder and not affecting its ranked status. This is currently not possible in lazer afaik, and idk if the developers would like to add that back

**Reply by Floatyoats** [reply]
ah okay I made this idea suggestion cause it was kinda annoying to delete the bg of every map I downloaded but if It's not possible that's all good

**Reply by felipemarins** [reply]
> but if It's not possible that's all good

If what you're asking for is to disable the map background on song select, it's entirely possible, it just need to be implemented by someone and approved by the dev team. What cihe was saying that may not come back is the option to modify the background file without unranking the beatmap you have. But you wouldn't need to modify the background file if someone was to implement a proper way to disable the map background on song select. Basically it would just need to hide the background.
Song select will be redesigned in the future, so I think you will need to wait until then to implement something like that.

**Reply by Floatyoats** [reply]
oh I see yeah that's exactly what I was saying just something simple like a black box that covers the backgrounds on the song select screen it would be nice if it also hid them when loading into a map but I could see how that wouldn't be doable

**Reply by cihe13375** [reply]
oh if that is the goal also see https://github.com/ppy/osu/discussions/20950 and https://github.com/ppy/osu/discussions/13333 I guess
